openapi/components/schemas/SubscriptionChange.yaml

```
@@ -25,6 +25,62 @@ properties:
         quantity:
           description: Number of units of the product on the given plan.
           type: integer
+        usageLimits:
+          type: object
+          description: |-
+            Specifies the soft and hard usage limits thresholds for an item with a metered billing plan.
```

nitpick: I think 'limit' should be singular here
```suggestion
            Specifies the soft and hard usage limit thresholds for an item with a metered billing plan.
```

---

Yes, I think so. 'Usage limit thresholds' could contain multiple limits. 'usage limits thresholds' is difficult to understand